source/common/network/udp_listener_impl.cc

```diff
@@ -69,20 +69,18 @@ void UdpListenerImpl::onSocketEvent(short flags) {
 void UdpListenerImpl::handleReadCallback() {
   ENVOY_UDP_LOG(trace, "handleReadCallback");
   // Max UDP payload.
-  constexpr uint64_t const read_buffer_length = 1500;
+  const uint64_t read_buffer_length = 1500;
```

Comment:
Sorry I meant here to make the 1500 a constant with a defined name, not switch from constexpr, and also to at least add some comments on how we might support jumbo frames in the future.

Reply:
Added MAX_UDP_PACKET_SIZE at the beginning of this file and TODO here to load this variable from config. Hope I understood correctly this time.